### DISCUSSION TITLE
Upgrade from trail to enterprise

### DISCUSSION THREAD

**Discussion by ARVITsecond** [community/community#147279] [Enterprise]
### Select Topic Area

Question

### Body

Currently, our account is using the enterprise trial version, but we have already purchased the license. However, we are unable to upgrade our account to Enterprise and cannot find a solution for the upgrade. Could you please provide a recommendation?

not have download license on menu

**Comment by queenofcorgis** [comment]
Hi @ARVITsecond ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by ARVITsecond** [reply]
Thank you very much for your suggestion. I have already created a ticket on the support portal, but the response has been very slow, so I will try another channel for assistance.
